### ISSUE TITLE
Arranging hotels/accommodation near the venue

### ISSUE THREAD

**Issue by jinalfoflia** [sotm-asia/sotm-asia-2018#18]
We need to find accommodation[s] that could be recommended to the delegates attending the conference. It would be great if could get these rooms at a discounted rates. We could _maybe_ offer them to be our accommodation partner or so.

I did a quick skim through via Google and found these that are closer to the venue: [spreadsheet](https://docs.google.com/spreadsheets/d/1WX8bzqKzA-Vo-KJt2er05rYznjTLSpPFYz_T1Juh6ws/edit?usp=sharing)

Please add more suggestions/recommendations. It'll be awesome if someone could jump in and help us run this out. 

cc: @geohacker

**Comment by poornibadrinath** [comment]
@jinalfoflia @geohacker I feel that since JP nagar is close to IIM Bangalore, we should look for accommodation there instead of somewhere close to IIM B. There are not many good staying options there. The only downside to having hotels in JP Nagar is the travel. 

Also, @jinalfoflia I can help out a bit with this. A little free on the coming weekends. What needs to be done??

**Comment by jinalfoflia** [comment]
>Also, @jinalfoflia I can help out a bit with this. A little free on the coming weekends. What needs to be done??

@poornibadrinath I agree, there might be better hotels in JP Nagar, no harm in checking them out too. 
I'm adding the hotels that I come across in the spreadsheet, if you can also search add the details to the spreadsheet, we can shortlist a few and post that go and negotiate with them.

**Comment by geohacker** [comment]
Thank you for getting started on this one @jinalfoflia @poornibadrinath! JP Nagar / Jayanagar may not be bad either. We're considering organising buses to bring the participants to the venue in the morning so that might work well.

**Comment by geohacker** [comment]
When you come up with a shortlist, we should call them and find out bulk booking discounts.

**Comment by jinalfoflia** [comment]
@poornibadrinath did you get a chance to update the list of hotels? I have some time this weekend, will add more to the list.

**Comment by poornibadrinath** [comment]
I have added a few hotels in the vicinity to the list. We need to call or go there and check out the prices and stuff. 

The hotel list is [here](https://docs.google.com/spreadsheets/d/1WX8bzqKzA-Vo-KJt2er05rYznjTLSpPFYz_T1Juh6ws/edit#gid=0)

I am still looking for a few hotels in the area. Booking.com has a huge list of places. Will check and update accordingly. 

@jinalfoflia we can check a few this weekend if that works :)

**Comment by poornibadrinath** [comment]
Sorry for the delay here, some mix ups with numbers and emails and slow responses. I have updated a few details in the spreadsheet [here](https://docs.google.com/spreadsheets/d/1WX8bzqKzA-Vo-KJt2er05rYznjTLSpPFYz_T1Juh6ws/edit#gid=0). Most of the hotels listed here have availability on the dates 16-19 of November. I called and enquired all of the hotels, they all have availability and the room rates are around the rates mentioned in the spreadsheet. 

For bulk booking enquiries, emails have been initiated and I am waiting for their response on quotes of the room tariff and how many final rooms are available in each hotel. Out of all the hotels, Pai group of hotels and Treebo Lalbagh inn have more rooms available on those particular days and are willing to immediately block the rooms for these days. Following up with them on emails, will update the spreadsheet with the final count of rooms and the discount rates within the next week. Once the final quotes are listed, we can go ahead and block the number of rooms required based on the number of attendees. 

Will post the update soon on the final numbers.

**Comment by geohacker** [comment]
@poornibadrinath this is great start. Thank you. Could you add another column to the sheet about the distance and how people would commute? 

@poornibadrinath can you also followup with them and arrive at a price/discount this week? Thank you!

**Comment by poornibadrinath** [comment]
> @poornibadrinath this is great start. Thank you. Could you add another column to the sheet about the distance and how people would commute?

Sorry for the delay here. Yes, I will add the column and the details by tomorrow. 

> @poornibadrinath can you also followup with them and arrive at a price/discount this week? Thank you!

Waiting for a few more emails from all of the hotels, will have an update ready by the end of the week and we can move ahead and block the rooms.

**Comment by poornibadrinath** [comment]
Getting ready the spreadsheet with all the details of the hotels and tariffs. Will post the update by eod tomorrow. Sorry for the delay here, was a bit busy with various priorities and was waiting for a few more replies.

**Comment by poornibadrinath** [comment]
I have added the distance column in the spreadsheet. These hotels are located in the arrange of 3-7 kms IIM Bangalore. The nearest ones from IIMB are Euphoria, Aarama Suites and The Grand Continent. Out of which, I haven't been able to get in touch with Euphoria at all. The number and the email are both faulty. However, I will try to find an alternative way of contact and find out how many rooms are available with them. 

The rest of them, I have updated the details [here](https://docs.google.com/spreadsheets/d/1WX8bzqKzA-Vo-KJt2er05rYznjTLSpPFYz_T1Juh6ws/edit#gid=0) in the spreadsheet on special tariffs. Still need to hear from a couple of hotels on their discounted rates. But the actual tariffs are updated and all of the hotels now have availability for bulk bookings. 

Kandi tree, Lilac and SFO suites are more luxurious than the rest of them. In case someone wants that option, these three hotels can be suggested. I don't think we necessarily need bulk booking options for these three hotels because not everyone will opt for a pricey hotel. 

Pai hotels and Treebo Lalbagh have the most rooms available and I have added the tentative rates for single and double occupancy for each of these hotels. @geohacker @jinalfoflia , the hotels require a final number of how many rooms we want in order for them to give a proper discount quote. For now, based on a ball park number of 25-30 rooms, the tariffs have been updated in the spreadsheet. 

I am in contact with all of these hotels, I will update as and when I get more information in the spreadsheet. 

Out of all these hotels, I am suggesting we go for Treebo and Pai hotels for their affordable prices and since they are closer to each other, it will be easier to arrange transport to and fro to iimb. It would be great if I can get a final number of how many rooms we are looking to book so we can start the process of blocking rooms.
